core/embed/rust/src/ui/model_tt/component/mod.rs

```diff
@@ -8,6 +8,8 @@ mod number_input;
 mod page;
 mod scroll;
 mod swipe;
+mod webauthn;
+pub mod webauthn_icons;
```

Comment:
I think this doesn't need to be pub?

Reply:
Yeah, it should not differ from the rest without a big reason.

Question is whether to even have `webauthn_icons` module, we could theoretically define all of that in `webauthn`, it should not be used anywhere else. Do you have some preference?

Reply:
`pub`deleted in [b557d76](https://github.com/trezor/trezor-firmware/pull/2556/commits/b557d76468ea90567cba981b4288e96502a5b73f)

Reply:
I'd keep it in the separate file. Sorry for the insufficient explanation, what I had in mind is this:

* delete the `pub use webauthn_icons::get_webauthn_icon_data;` line in mod.rs as we're not using the function outside of the module
* in webauthn.rs import the function from `crate::ui::model_tt::component::webauthn_icons::get_webauthn_icon_data`

Reply:
Done like this in [f44f512](https://github.com/trezor/trezor-firmware/pull/2556/commits/f44f5126ddf39a27ba035f698f2512abf0ea72c8)